source/ProSymbolEditor/ViewModels/MilitarySymbolDockpaneViewModel.cs

```diff
@@ -500,8 +631,18 @@ public SelectedFeature SelectedSelectedFeature
 
                 if (_selectedSelectedFeature != null)
                 {
-                    MapView.Active.FlashFeature(_selectedSelectedFeature.FeatureLayer, _selectedSelectedFeature.ObjectId);
-                    CreateSymbolSetFromFieldValuesAsync();
+                    try
+                    {
+                        // TODO: there is an exception here when:
+                        // 1: Multiple Maps are open
+                        // 2: Trying to flash a feature that is selected on another map, that is not the active map
+                        MapView.Active.FlashFeature(_selectedSelectedFeature.FeatureLayer, _selectedSelectedFeature.ObjectId);
+                        CreateSymbolSetFromFieldValuesAsync();
+                    }
+                    catch (Exception exception)
+                    {
+                        System.Diagnostics.Debug.WriteLine(exception.ToString());
```

Comment:
I know this is for debugging purposes but should be removed. Use another way to capture exception and report to user.